ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -86,7 +90,8 @@ node('caasp-team-private-integration') {
            if (pr_repo_label != null) {
                def branch_name = pr_repo_label.name.split(":")[1]
                branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/${repo_version}:/Branches:/${branch_name}/SLE_15_SP2"
-
+               branch_registry = "registry.suse.de/Devel/CaaSP/${repo_version}/Branches/${branch_name}/containers/cr/containers/caasp/v${repo_version}"
+               original_registry = "registry.suse.de/devel/caasp/5/containers/cr/containers/caasp/v5"
```

Comment:
For consistency with cri-o terminology, the `original_registry` should be `registry_prefix` but I will not stop the PR for this. Same with environment variables.